### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by dialelo**

### CODE REVIEW

**Comment by dialelo** [comment]
Addresses #55

File: build.sbt [hunk 1]
```diff
@@ -1,19 +1,14 @@
 import de.heikoseeberger.sbtheader.license.MIT
-import catext.Dependencies._
 import PgpKeys.gpgCommand
 
-val dev  = Seq(Dev("47 Degrees (twitter: @47deg)", "47 Degrees"))
-val gh   = GitHubSettings("com.fortysevendeg", "github4s", "47 Degrees", mit)
```
Comment (juanpedromoreno) [inline build.sbt]: We still need these settings for releasing purposes. Can we get these back?

File: build.sbt [hunk 2]
```diff
@@ -32,8 +27,7 @@ lazy val buildSettings = Seq(
     sharedCommonSettings ++
     miscSettings ++
     sharedReleaseProcess ++
-    credentialSettings ++
-    sharedPublishSettings(gh, dev)
```
Comment (juanpedromoreno) [inline build.sbt]: Same here, `sharedPublishSettings` is needed.

**Comment by dialelo** [comment]
@juanpedromoreno just added the publish settings back, let's wait for Travis to give us :green_heart: and we can merge this.

**Pull request merged**